backend/src/main/resources/graphql/main.graphqls

```diff
@@ -59,9 +59,18 @@ type DeviceType {
   model: String
   loincCode: String
   swabType: String
+  swabTypes: String
```

Comment:
this was added due to the following error
`[{"message":"Validation error of type FieldUndefined: Field 'swabTypes' in type 'DeviceType' is undefined @ 'createDeviceTypeNew/swabTypes'","locations":[]}]`